students/morgan/session01/break_me.py

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
I'm not sure why this file is here. It is in someone else's directory, and it doesn't look different from the one that is in the repo. I assume you must have done a git pull somewhat recently. I think something must have gone weird with a merge. I'm going to leave it as is, and see how it goes...